app/routes/image-sets/new.js

```diff
@@ -1,12 +1,19 @@
+import Ember from 'ember';
 import ImageSetRoute from '../image-set';
+import {defaultLocation} from 'lion-guardians/utils/units';
 
 export default ImageSetRoute.extend({
   controllerName: 'image-set',
+  defaultLocation: defaultLocation,
 
   model: function() {
-    var currentUser = this.get('toriiSession.currentUser');
+    var currentUser = this.get('toriiSession.currentUser'),
+        defaultLocation = this.get('defaultLocation');
+
     return this.store.createRecord('image-set', {
-      user: currentUser
+      user: currentUser,
+      latitude: Ember.get(defaultLocation, 'lat'),
```

Comment:
it would be safe to do `defaultLocation.lat` since we know and can control what `defaultLocation` is. `Ember.get(object, propertyPath)` is mostly for cases where we do not control whether an `object` is an Ember Object or a pojo (i.e., if it were the input to a function). No harm in `Ember.get`-ing it here, but not necessary either.